lochness/email/__init__.py

```diff
@@ -195,16 +196,19 @@ def send_out_daily_updates(Lochness, days: int = 1,
 
         send_detail(
             Lochness,
+            Lochness['sender'],
+            Lochness['notify'],
```

Comment:
Please double-check whether the above two `Lochness['sender']` should be just `sender` that you passed to this function newly.

Reply:
Never mind!